src/main/webapp/app/tutor-exercise-dashboard/tutor-exercise-dashboard.component.html

```diff
@@ -373,9 +380,12 @@ <h4>
                     </table>
 
                     <ng-template #noComplaints>
-                        <div class="alert alert-info">
+                        <div *ngIf="numberOfMoreFeedbackRequests === 0" class="alert alert-info">
                             {{ 'artemisApp.tutorExerciseDashboard.noMoreFeedbackRequests' | translate }}
                         </div>
+                        <div *ngIf="numberOfMoreFeedbackRequests != 0" class="alert alert-info">
```

Comment:
```suggestion
                        <div *ngIf="numberOfMoreFeedbackRequests !== 0" class="alert alert-info">
```

Reply:
not sure if this would be more correct, but this would be more consistent to the `===` above

Reply:
yes, that's definitely better to be more consistent. Than you!